routes/actions/actionsRouter.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const db = require("../../data/helpers/actionModel");
+const dbProjects = require("../../data/helpers/projectModel");
+
+const router = express.Router();
+
+//get all actions
+router.get("/", async (req, res) => {
+  try {
+    const allActions = await db.get();
+    res.json({ message: allActions });
+  } catch (err) {
+    res.status(500).json({ message: "internal server erros" });
+  }
+});
+
+//get action by ID
+router.get("/:id", validateID, async (req, res) => {
+  try {
+    const singleAction = await db.get(req.params.id);
+    res.json({ message: singleAction });
+  } catch (err) {
+    res.status(500).json({ message: "internal server erros" });
+  }
+});
+
+//add action
+router.post("/", async (req, res) => {
+  try {
+    const validateProject = await dbProjects.get(req.body.project_id);
```

Comment:
Great job with validation, shows you're really thinking about how an API should work instead of just completing MVP points